pkg/api.go

```diff
@@ -12,6 +13,7 @@ import (
 	"github.com/hyperledger/aries-framework-go/pkg/kms"
 )
 
+// KeyHandleWriter defines key handler writer APIs.
```

Comment:
For now added basic comments to fix lint issues. These need to be updated in future PRs.